docs/docusaurus/docs/cloud/quickstarts/snowflake_quickstart.md

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_label: 'Quickstart for GX Cloud and Snowflake'
+title: 'Quickstart for GX Cloud and Snowflake'
+id: snowflake_quickstart
+description: Connect Great Expectations to Snowflake Data Assets.
+---
+
+In this quickstart, you'll learn how to connect GX Cloud to Snowflake Data Assets.
+
+## Prerequisites
+
+- You have a [GX Cloud Beta account](https://greatexpectations.io/cloud).
+
+- You have a [Snowflake account](https://greatexpectations.io/cloud) with `ACCOUNTADMIN` access.
+
+## Prepare your environment
+
+1. Download and install the latest Python version. See [Active Python Releases](https://www.python.org/downloads/).
+
+2. Download and install pip. See the [pip documentation](https://pip.pypa.io/en/stable/cli/pip/).
+
+3. Run the following command in an empty base directory inside a Python virtual environment to install GX Cloud and its dependencies:
+
+    ```bash title="Terminal input"
+    pip install great_expectations[cloud]
+    ```
+
+    It can take several minutes for the installation to complete.
+
+    If you've previously installed GX Cloud, run the following command to upgrade to the latest version:
+
+    ```bash title="Terminal input"
+    pip install great_expectations[cloud] --upgrade
```

Comment:
```suggestion
    pip install 'great_expectations[cloud]' --upgrade
```
This will fail without these quotes on `ZSH` which is the default shell on MacOS.